.rubocop.yml

```diff
@@ -11,13 +9,16 @@ AllCops:
     - 'actionpack/lib/action_dispatch/journey/parser.rb'
     - 'railties/test/fixtures/tmp/**/*'
 
-# Prefer assert_not_x over refute_x
-CustomCops/RefuteNot:
+Rails:
+  Enabled: true
```

Comment:
We don't want to enable a whole category of checks here -- we should just enable the individual ones we want.

Reply:
AFAIK, the behavior of `Rails` department is special.

`Rails` department is disabled by default. And enabling `Rails` department does not enable all `Rails` cops. So only specified `Rails` cops will be enabled. Here `Rails/AssertNot` cop and `Rails/RefuteMethods` cop are enabled.

The following is quoted from the official document.
https://github.com/rubocop-hq/rubocop/blob/v0.57.2/manual/configuration.md#enabled

> The exception to the rule is the `Rails` department, which can not be enabled
> in its entirety this way. Setting `Rails: Enabled: true` will have the same
> effect as running with the `--rails` command line option, which in the context
> of `DisabledByDefault: true` means to make it possible to enable Rails cops
> individually.
>
> ```yaml
> AllCops:
>   EnabledByDefault: true
> ```